go.mod

```diff
@@ -1,6 +1,6 @@
 module knative.dev/pkg
 
-go 1.22
+go 1.22.0
```

Comment:
@dprotaso is there any specific reason for the go patch version in `go` directive? Asking for the silent error in `update-deps` run[1]. Or any other potential side effects?

[1] https://github.com/knative-extensions/knobots/actions/runs/10352426965/job/28653073330#step:6:430

Reply:
@dsimansk the go toolchain was forcing it - I couldn't figure out how to let it remain 1.22

Reply:
Yep, it's force through k8s.io modules. There's top level go.work with `go 1.22.0` that's reflected through toolchain `go.mod` upadate behavior to every importing module as well.

I guess, we won't be able to get rid off for a while. 

https://github.com/kubernetes/kubernetes/blob/release-1.30/staging/src/k8s.io/client-go/go.mod#L5